scripts/yalc-publish.sh

```diff
@@ -0,0 +1,20 @@
+chmod +x ./scripts/yalc-publish.sh
+package='yalc'
+if [ `npm list -g | grep -c $package` -eq 0 ]; then
+    npm install -g $package --no-shrinkwrap
+fi
+
+if [ ! $1 ]; 
+then
+  printf "If you want to publish a specific package, use the folder name as argument. \nExample: (yarn yalc-publish core).\n"
```

Comment:
```suggestion
  printf "If you want to publish a specific package, use the folder name as argument. \nExample: `yarn yalc-publish core`.\n"
```

Feels more natural, giving an example in parenthesis itches me 😛  (the same for the other script)

Reply:
In that way it will be presented like this: `Example: $ scripts/yalc-publish.sh core`